molo/core/tests/base.py

```diff
@@ -266,6 +267,8 @@ def mk_articles(self, parent, count=2, **kwargs):
                 'slug': generate_slug(data['title'])
             })
             article = ArticlePage(**data)
+            if not article.first_published_at:
+                article.first_published_at = datetime.now()
```

Comment:
did this make a difference the tests? I am wondering what triggered this requirement

Reply:
@sewagodimo there's nothing in the PR that I can see that _should_ have made this necessary. but we've been encountering the same problem you had where articles created in the tests didn't have a `first_published_at` set.